change hardcoded $HOME/.luscus path to config subdir of installation ($LUSCUS_DIR should be set to point there)
author: Kenneth Hoste (HPC-UGent)
--- luscus_0.8.6/CMakeLists.txt.orig	2019-05-09 20:54:35.712120873 +0200
+++ luscus_0.8.6/CMakeLists.txt	2019-05-09 20:55:06.922437408 +0200
@@ -31,7 +31,7 @@
     set(CONFIG_DIR "/etc/luscus")
 #    message(status " CMAKE_PREFIX_PATH NOT DEFINED!")    # DEBUG
   else ()
-    set(CONFIG_DIR "$ENV{HOME}/.luscus")
+    set(CONFIG_DIR "${CMAKE_INSTALL_PREFIX}/config")
 #    message(status " CMAKE_PREFIX_PATH DEFINED!")        # DEBUG
   endif ()
   set(TMP_CONFIG_DIR ${CMAKE_CURRENT_BINARY_DIR}/luscusrc)
